datadisplay.cpp

```diff
@@ -59,6 +59,9 @@ DataDisplay::DataDisplay(QWidget *parent)
 
     connect(m_searchPanel, &SearchPanel::findNext, this, &DataDisplay::find);
     connect(m_searchPanel, &SearchPanel::textEntered, m_highlighter, &DataHighlighter::setSearchString);
+
+    connect(&m_timer, SIGNAL(timeout()), this, SLOT(BlockReady()));
+    m_timer.start(100);
```

Comment:
When you do rebase that means you are rewriting the history. If you will push that onto GitHub then all old commits and theirs inline comments will disappear, so please make an another commit.